exporter/exporter.go

```diff
@@ -179,7 +181,7 @@ func (x *Exporter) update() {
 	if !start.IsZero() {
 		x.onceRegisterUpdateRunning.Do(func() { prometheus.MustRegister(x.promUpdateRunning) })
 		if end.IsZero() || start.After(end) {
-			log.Println("An update run started.")
+			x.log.Debugf("An update run started.")
```

Comment:
Debugf is proprietary to logrus. Since there are only two cases (debug, or no debug), consider using a simple wrapper that tests for the debug flag.